react/PickupPointsModal.js

```diff
@@ -173,6 +174,11 @@ class PickupPointsModal extends Component {
     this.props.onAddressChange(addressValidated)
   }
 
+  handleManuallyCloseModal = () => {
```

Comment:
The `handleCloseModal` handler is also used when the modal has been closed due to the pickup point being selected and confirmed. This handler specifically handles the case where the user clicks on the close button or on the backdrop (i.e. when the search hasnt been satisfactory, probably)